tox.ini

```diff
@@ -14,16 +14,16 @@ commands = python -m pip install --upgrade pip
 # the tests environment is called by the Github action that runs the unit tests
 deps =
     -r requirements.txt
-    -r dev_requirements/requirements-tests.txt
-    -r dev_requirements/requirements-extra-aiostream.txt
+    .[tests]
+    .[aiostream]
```

Comment:
Du musst noch die requirements neu kompilieren, die alte (requirements.txt) löschen und dann würde ich an dieser Stelle die requirements installieren, da `seviper[aiostream]` (korrekterweise) keine gepinnten Dependencies hat.

Reply:
Dafür brauche ich dann den befehl

Reply:
oder meintest ud das mit pip-compile pyproc
511361134cd67460e46defb85aaed01a6cfb1f8f

Reply:
Done in https://github.com/Hochfrequenz/seviper/pull/58/commits/a654d35be145784b83dd504fd92d5d09c4a742d4